### DISCUSSION TITLE
Coordinator for tinkerers - EFR32MG24 / XG24-EK2703A - ezsp

### DISCUSSION THREAD

**Discussion by Wireheadbe** [Koenkk/zigbee2mqtt#18046] [Show and tell]
For those who want to play around with the EZSP platform (and possibly make a nice and powerful device..), there's a nice reference board by Silicon Labs, which costs about 38 EUR, which gives you:

- A **reference** implementation of the EFR32MG24 wireless SoC (so **_known good_** hardware/implementation)
- Bluetooth LE, Bluetooth mesh, Zigbee, Thread, and Matter compatibility
- Can be connected to a good external antenna (e.g. ANT-2.4-PW-QW-UFL at Mouser.com), but comes with an on-board chip antenna. To use an external antenna, one needs to relocate a 18pF capacitor.
- Allows you to roll your own firmware, by means of the free Simplicity Studio

You can make this firmware in Simplicity studio by compiling two projects:

- bootloader-uart-xmodem: a bootloader for the board
- ncp-uart-hw: network-Co-Processor, which turns the device into a Zigbee Coordinator

I've also designed a small snap-fit enclosure for it, for those who want to have a go with it - you can find it at [Printables](https://www.printables.com/model/507127-xg24-ek2703a-enclosure)

![gecko02](https://github.com/Koenkk/zigbee2mqtt/assets/4497972/1828afa5-d901-4416-bc7a-8eedd4461e0d)

**Comment by mercenaruss** [comment]
@Wireheadbe i have couple of boards with EDR32xG24 on board,i just run now simplicity studio and try to upload the firmware( i dont have experience) but no luck,is building,everyhting fine,but i miss something.Can you give more details on this?

**Reply by Wireheadbe** [reply]
Did you upload the bootloader as well? 😄

**Reply by mercenaruss** [reply]
Okay,now in Simplicity Studio Bootloader is available. Can you direct me on this?

**Comment by jakymiwm** [comment]
You need to move CC1 to CC2.
![Screenshot Caps](https://github.com/user-attachments/assets/9ee2d0aa-0797-4c52-bcaa-cc5f5b346d02)
